scripts/completion/dvc.bash

```diff
@@ -58,23 +58,118 @@ _dvc_version=''
 #       x="hello"
 #       ${!x} ->  ${hello} ->  "world"
 #
-_dvc () {
-  local word="${COMP_WORDS[COMP_CWORD]}"
+_dvc ()
+{
+    local cur prev opts
+            COMPREPLY=()
```

Comment:
Also, what is this additional indent is about in this whole code block?
```suggestion
    COMPREPLY=()
```

Reply:
Sorry about indentation, I haven't checked them.